### PR TITLE
Fix fl model utils

### PR DESCRIPTION
**Pull request by YuanTingHsieh**
### Description

Use explicit argument name instead of kwargs

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Quick tests passed locally by running `./runtest.sh`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated.

### CODE REVIEW

File: nvflare/app_common/utils/fl_model_utils.py [hunk 1]
```diff
@@ -104,36 +103,42 @@ def from_shareable(
...
+            meta[MetaKey.JOB_ID] = fl_ctx.get_job_id()
+            meta[MetaKey.SITE_NAME] = fl_ctx.get_identity_name()
+
+        result = FLModel(
+            params_type=params_type,
+            params=params,
+            metrics=metrics,
+            current_round=current_round,
+            total_rounds=total_rounds,
+            meta=meta,
```
Comment (yhwen) [inline nvflare/app_common/utils/fl_model_utils.py:140]: Should we treat the current_round and total_rounds part of the "meta"?

Comment (YuanTingHsieh) [reply nvflare/app_common/utils/fl_model_utils.py:140]: I am neutral on this, @holgerroth @chesterxgchen what do you think?

**Pull request merged**